packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -137,10 +135,9 @@ async function createIncomingPayment(
       paymentPointerId,
       client,
       assetId: paymentPointer.asset.id,
-      description,
       expiresAt,
       incomingAmount,
-      externalRef,
+      metadata,
```

Comment:
I thought we might need to handle the serialization of arrays here due to this issue described in the `knex` docs about json/jsonb and postgres arrays (https://knexjs.org/guide/schema-builder.html#json). Here is an example of this problem in the wild: https://github.com/knex/knex/issues/1349

However I did some testing and this wasn't required - I presume because we are using `Objection` and not `knex` directly. I included an array in the incoming/outgoing payment metadata tests and inspected the db value manually to confirm.